### ISSUE TITLE
Connection reset by peer

### ISSUE THREAD

**Issue by iuliancioarca** [plotly/Dash.jl#151]
I keep getting this warning when deploying a Dash app.

>  Warning: connection reset by peer (ECONNRESET)
└ @ HTTP.Servers ~/home/.julia/packages/HTTP/qszg7/src/Servers.jl:309

Recently I discovered that one of my callbacks does not update some FE widgets when this warning occurs.
I suspect it's related to some timeout in the server. Can it be adjusted? How can I fix this?

**Comment by waralex** [comment]
Hi!
I'll try to figure out what's going on. If there is an opportunity to look at your code, then it can help a lot. Perhaps the problem is in the long compilation of the first call of some callback. I do not promise that it will be possible to fix the problem quickly - perhaps it is on the HTTP.jl side

**Comment by iuliancioarca** [comment]
I apologize, I've rushed a bit with this issue. The problem was on the cloud side. The app was deployed using OpenShift and I had to manually configure the route timeout.
I also looked into the Dash/HTTP code. The `serve` method has a default timeout kwarg:
https://github.com/JuliaWeb/HTTP.jl/blob/10f408c54be48187f2c0cfd5bb47cbb297ed312d/src/Handlers.jl#L349
as well as the underlying `listen` method. The `readtimeout` kwarg has already a default value of 0, meaning timeout disabled. So from this perspective Dash doesn't have to take care of timeouts.

Thank you for the swift response!
